docs/src/pages/components/snackbars/ConsecutiveSnackbars.js

```diff
@@ -12,88 +12,80 @@ const styles = theme => ({
   },
 });
 
-class ConsecutiveSnackbars extends React.Component {
-  queue = [];
+function ConsecutiveSnackbars(props) {
+  const queue = [];
```

Comment:
Should this be a ref?

Reply:
Not sure if it should be a ref, but it should be in state IMO

Reply:
Rule of thumb: if it was accessed via `this.field` in a class component it should be `fieldRef` in a function component.

Anything else would be a change in behavior which should be avoided when "just" migrating.